README.md

```diff
@@ -165,6 +164,7 @@ To the contributors on GitHub:
 - paper42
 - phpwutz
 - rapenne-s
+- rdbende
```

Comment:
Apparently, `Adw.AboutWindow` has a method (`new_from_appdata`) to pull the metadata and release notes from the appdata file, so that would be nice, but that's for another PR.

Reply:
I'll do this in a separate PR